src/routes/board/index.tsx

```diff
@@ -0,0 +1,6 @@
+import { component$ } from "@builder.io/qwik";
+import { GameBoard } from "~/components/game-board/board";
+
+export default component$(() => {
+	return <GameBoard />;
```

Comment:
[**A personal preference** (PR comments with a purpose)](https://gist.github.com/dgoings/ecfe4877b5992190e4fc8893c77a7ae6):

Is this prematurely abstracted? Maybe inline this component? Talked it over, leaving for now but let's keep an eye on it as the app is developed.